packages/resource-deployment/templates/on-demand-scan-schedule.template.json

```diff
@@ -1,5 +1,5 @@
 {
-    "id": "url-web-api-scan-schedule",
+    "id": "on-demand-scan-schedule",
```

Comment:
why it's not "on-demand-url-scan-schedule", especially we the task name has "url" keyword in it